pkg/client/client.go

```diff
@@ -257,69 +305,68 @@ func (c *Client) ShowTables() ([]string, error) {
 	return tables, nil
 }
 
-// NextPage returns the next page of the given table, based off the limit and the offsite.
-func (c *Client) NextPage() (*Table, int, error) {
-	if err := c.paginationManager.NextPage(); err != nil {
-		return nil, 0, err
-	}
+// ShowTables list all the tables in the database on the tables panel.
+func (c *Client) ShowTables() ([]string, error) {
+	var (
+		query string
+		err   error
+		args  []interface{}
+	)
+
+	tables := make([]string, 0)
 
-	r, col, err := c.tableContent(c.paginationManager.CurrentTable())
+	query, args, err = c.databaseQuerier.ShowTables()
 	if err != nil {
-		return nil, 0, err
+		return nil, err
 	}
 
-	t := Table{
-		name:    c.paginationManager.CurrentTable(),
-		Rows:    r,
-		Columns: col,
+	rows, err := c.db.Queryx(query, args...)
+	if err != nil {
+		return nil, err
 	}
 
-	page := c.paginationManager.CurrentPage()
-
-	return &t, page, nil
-}
+	for rows.Next() {
+		var table string
+		if err := rows.Scan(&table); err != nil {
+			return nil, err
+		}
 
-// PreviousPage returns the next page of the given table, based off the limit and the offsite.
-func (c *Client) PreviousPage() (*Table, int, error) {
-	if err := c.paginationManager.PreviousPage(); err != nil {
-		return nil, 0, err
+		tables = append(tables, table)
 	}
```

Comment:
Each row.Nect loop must be followed by checking rows.Err()

```suggestion
	}

 	if err:= rows.Err(); err != nil {
 	 	return nil, err
 	}
```